scripts/wait-for-npm-package.sh

```diff
@@ -1,21 +1,38 @@
 #!/usr/bin/env bash
```

Comment:
Why do we need this script? I thought we were waiting for the NPM package registry

Reply:
correct, that is what this script is doing